microservices/first/utils/db_manager.py

```diff
@@ -1,120 +1,134 @@
 import sqlite3
-from typing import Union
+from flask import g
+import json
 
 from . import deployment_status
 
 """
 Deployment Table Schema:
 id: Integer, Primary key identifying each row
-leader_ID: Integer, Indentifier of deployment
+leader_ID: TEXT, Indentifier of deployment
 repo_name: TEXT, Name of repo the deployment is working on
 repo_branch: TEXT, Branch of the repo being worked on
 status: TEXT, (DeploymentStatus in text form) Status of the deployment.
 
 Results Table Schema:
 id: Integer, Primary key identifying each row
-leader_ID: Integer, Foreign Key pointing to Deployment Table "leader_id". Deployment the result is associated with.
+deployment_ID: Integer, Foreign Key pointing to Deployment Table "id". Deployment the result is associated with.
```

Comment:
same here

Reply:
This was initially leader_ID and "Text" but claimed to be a foreign key pointing to ID in the deployment table which was integer (the deployment ID).

Reply:
Essentially while they were written as "leader_ID" by syntax, they were functionally used as deployment IDs